apps/finance/contracts/Finance.sol

```diff
@@ -15,7 +16,7 @@ import "@aragon/apps-vault/contracts/IVaultConnector.sol";
 import "@aragon/os/contracts/lib/misc/Migrations.sol";
 
 
-contract Finance is AragonApp {
+contract Finance is IsContract, AragonApp {
```

Comment:
Note that solidity fails to linearize the inheritance tree if it's `AragonApp, IsContract`.